actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java

```diff
@@ -100,7 +100,9 @@ private void setSystemSetting(Request actorMessage) {
     if (JsonKey.PHONE_UNIQUE.equalsIgnoreCase(field)
             || JsonKey.EMAIL_UNIQUE.equalsIgnoreCase(field)
             || JsonKey.PHONE_UNIQUE.equalsIgnoreCase(id)
-            || JsonKey.EMAIL_UNIQUE.equalsIgnoreCase(id)) {
+            || JsonKey.EMAIL_UNIQUE.equalsIgnoreCase(id)
```

Comment:
Can we make it more generic:
Example: define all valuses inside externalproperties file with comma separated, and make list from that value.
same can be set by adopter inside env if they want to allow set or update using api